Models/Comment.cs

```diff
@@ -1,12 +1,17 @@
-using System
+using System;
 
 
-namespace Kumustagram_API.Models.Comment
+namespace Kumustagram_API.Models
 {
     public class Comment
     {
-        private int Id { get; set; }
-        private int user {  get; set; }
-        private string comment { get; set; }
+        public int Id { get; set; }
+        public int User { get; set; }
+        public string CommentStr { get; set; }
```

Comment:
It's okay to have long variable names.. It's much better when it's clear. 

Not sure what CommentStr means, is it CommentString?